app/src/main/java/com/github/se/signify/model/exercise/ExerciseInformation.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.se.signify.model.exercise
+
+import com.github.se.signify.R
+import com.github.se.signify.ui.navigation.Screen
+
+/**
+ * Data class that regroup information of the different level of exercise
+ *
+ * @param exerciseLevel Represent the level of the exercise based of an enum
+ *
+ * level : String to use for UI. levelRoute : Corresponding route of the level. wordsResourceId :
+ * Resource ID for the list of words or sentences to be used in the exercise. screenTag : Tag used
+ * for identifying the screen in tests. wordFilter : Optional filter function to specify criteria
+ * for selecting words from the resource.
+ */
+data class ExerciseInformation(val exerciseLevel: ExerciseLevel) {
+  val level =
+      when (exerciseLevel) {
+        ExerciseLevel.Easy -> "Easy"
+        ExerciseLevel.Medium -> "Medium"
+        ExerciseLevel.Hard -> "Hard"
+      }
+  val levelRoute =
+      when (exerciseLevel) {
+        ExerciseLevel.Easy -> Screen.EXERCISE_EASY
+        ExerciseLevel.Medium -> Screen.EXERCISE_MEDIUM
+        ExerciseLevel.Hard -> Screen.EXERCISE_HARD
```

Comment:
Maybe there is a way around having to define a screen for each exercise. But I can't find a good one that fits our codebase.